feature/gribi/otg_tests/gribigo_compliance_test/gribigo_compliance_test.go

```diff
@@ -225,3 +290,67 @@ func configureATE(t *testing.T, ate *ondatra.ATEDevice) gosnappi.Config {
 
 	return top
 }
+
+// AddNHGReferencingToUknownNH tests a case where a nexthop group references a nexthop IP that does not Resolve.
+func AddNHGReferencingToUnresolvedNH(tcArgs *testArgs, wantACK fluent.ProgrammingResult, t testing.TB) {
+	t.Log("Flush existing gRIBI routes before test.")
+	if err := gribi.FlushAll(tcArgs.client); err != nil {
+		t.Fatal(err)
+	}
+	tcArgs.client.Modify().AddEntry(t,
+		fluent.NextHopEntry().WithNetworkInstance(deviations.DefaultNetworkInstance(tcArgs.dut)).
+			WithIndex(2000).WithIPAddress("1.0.0.1"),
+		fluent.NextHopGroupEntry().WithNetworkInstance(deviations.DefaultNetworkInstance(tcArgs.dut)).
+			WithID(2000).AddNextHop(2000, 1),
+		fluent.IPv4Entry().WithNetworkInstance(deviations.DefaultNetworkInstance(tcArgs.dut)).
+			WithPrefix("203.0.113.101/32").WithNextHopGroup(2000))
+
+	if err := awaitTimeout(tcArgs.ctx, t, tcArgs.client, 2*time.Minute); err != nil {
+		t.Logf("Could not program entries via client, got err, check error codes: %v", err)
```

Comment:
Should this be a `t.Errorf` rather than a `t.Log`?

Reply:
this is fine we are printing the log only with and checking the below where actually it is failing/passing.

chk.HasResult(t, tcArgs.client.Results(t),